2017-12-30 21:42:27|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('useful', 'undefined', '60')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('useful', 'undefined', '60')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
2017-12-30 21:42:43|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('useful', 'undefined', '60')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('useful', 'undefined', '60')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
2017-12-30 21:42:43|ERROR|App.Comment.MinusOne|PDOException: Truncated incorrect DOUBLE value: 'undefined' in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(503): NotORM_Result->query('DELETE FROM com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(19): NotORM_Result->delete(1)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(55): App\Model\Useful->minusOne('useful', 'undefined', '60')
#3 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(78): App\Domain\Comment->minusOne('useful', 'undefined', '60')
#4 [internal function]: App\Api\Comment->minusOne()
#5 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#6 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#7 {main}
2017-12-30 21:42:44|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('useful', 'undefined', '60')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('useful', 'undefined', '60')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
